README.md

```diff
@@ -1,27 +1,76 @@
-# hm-config
-Helium Miner Config Container
+# hm-config: Helium Miner Config Container
```

Comment:
Maybe we should add in here a link to the upstream codebase that this is replicating....

https://github.com/helium/gateway-config